naga/src/front/wgsl/error.rs

```diff
@@ -144,7 +144,7 @@ pub enum InvalidAssignmentType {
 }
 
 #[derive(Clone, Debug)]
-pub enum Error<'a> {
+pub(crate) enum Error<'a> {
```

Comment:
issue: I know that some folks in the ecosystem have built tooling on top of `naga` by inspecting the concrete `Error` variants emitted by it. Is there a reason you're restricting this to `pub(crate)`?

Reply:
This type hasn't been exposed publicly. It's in a private module. 

It's not a change, but clarification of the existing behavior and a fix of misleading `pub` on a non-public type.